Samples/UIWidgetsSamples_2019_4/Assets/Editor/EditorWindowSample/GalleryMain.cs

```diff
@@ -9,10 +10,17 @@ public class EditorWindowGallery : UIWidgetsEditorPanel
     {
         [MenuItem("UIWidgets/EditorSample")]
         public static void CountDemo()
-        {
+        { 
             CreateWindow<EditorWindowGallery>();
         }
-        
+
+        protected override void loadConfiguration()
+        {
+            AddFont("Material Icons", new List<string>{"MaterialIcons-Regular.ttf"}, new List<int>{0});
+            AddFont("CupertinoIcons", new List<string>{"CupertinoIcons.ttf"}, new List<int>{0});
+            AddFont("GalleryIcons", new List<string>{"gallery/GalleryIcons.ttf"}, new List<int>{0,1});
```

Comment:
the length of assets is not the same as the length of weights on line 21, is it deliberately to be wrong?

Reply:
Yes, I listed it as a wrong example.